database/submit_to_database.py

```diff
@@ -570,95 +679,35 @@ def flux_cal_and_submit(time_detection, time_obs, metadata, bestprof_data,
             #TODO add a pointing for this
             fits_files_loc = '/group/mwavcs/vcs/{0}/pointings/*.fits'.format(args.obsid)
 
-    #get info from .bestprof file
-    if args.bestprof:
-        bestprof_data = prof_utils.get_from_bestprof(args.bestprof)
-        obsid, pulsar, dm, period, period_uncer, obsstart, time_detection,\
-               profile, num_bins = bestprof_data
-        if obsid is None and args.obsid:
-            obsid = args.obsid
-        elif obsid is None:
-            logger.error("Please use --obsid. Exiting")
-            sys.exit(1)
-    elif args.ascii:
-        profile, num_bins = prof_utils.get_from_ascii(args.ascii)
-        if args.obsid:
-            obsid = args.obsid
-        if args.pulsar:
-            pulsar = args.pulsar
-        dm, period = get_pulsar_dm_p(pulsar)
-        period_uncer = float(period) / 10.**10 #assuming it's miniscule
-        if args.stop and args.start:
-            time_detection = args.stop - args.start
-        else:
-            logger.error("Please use --start and --stop for ascii files. Exiting")
-            sys.exit(1)
-        bestprof_data = [obsid, pulsar, dm, period, period_uncer, args.start,
-                         time_detection, profile, num_bins]
```

Comment:
You've removed the ascii option here and looks like you haven't added an option to process them at line 707

Reply:
Fixed this but please double check my code